server/app/lib/utils/fusor/mac_discovery/dell.rb

```diff
@@ -0,0 +1,20 @@
+require 'rubyipmi'
+
+module Utils
+  module Fusor
+    module MacDiscovery
+      class Dell < Rubyipmi::Ipmitool::BaseCommand
+
+        def initialize(opts = ObservableHash.new)
+          super("ipmitool", opts)
+        end
+
+        def getmacs
+          @options["cmdargs"] = "delloem mac list"
+          runcmd
+          return result
```

Comment:
https://github.com/logicminds/rubyipmi#writing-a-function-for-running-a-command
Each subclassed baseCommand class inherits runcmd, result, cmd, and runcmd_with_args.

I don't remember what example I ran off of, but along the same lines:
https://github.com/logicminds/rubyipmi/blob/master/lib/rubyipmi/ipmitool/commands/fru.rb#L92